monai/transforms/composables.py

```diff
@@ -143,14 +128,7 @@ def __call__(self, data):
         d = dict(data)
         for key in self.keys:
             affine_key = self.meta_key_format.format(key, 'affine')
-            original_key = self.meta_key_format.format(key, 'original_affine')
-
-            affine = d.get(affine_key, None)
-            if affine is None:
-                affine = d.get(original_key, None)
-            d[key], affine_, new_affine = self.ornt_transform(d[key], affine)
-            if d.get(original_key, None) is None:
-                d[original_key] = affine_
+            d[key], _, new_affine = self.ornt_transform(d[key], original_affine=d[affine_key])
```

Comment:
I think you should change the argument name of orientation from "original_affine" to "affine" as it's confusing with the real "original_affine" in dict.

Reply:
that's a good point, I've revised all cases